### ISSUE TITLE
Collapse sidebar by default on the DToC objects 

### ISSUE THREAD

**Issue by ilovan** [cwrc/cwrc_dtoc_edition#10]
## Expected Behaviour

Default view:

![image](https://user-images.githubusercontent.com/6600070/52744555-3f996f80-2f9a-11e9-856d-8891ea3b109c.png)

## Current Behaviour

Default view:

![image](https://user-images.githubusercontent.com/6600070/52744526-2abcdc00-2f9a-11e9-845c-d5a7f9b1b77a.png)

## Possible Solution

This setting is enabled on dev-03, test-01, staging-01 and beta.cwrc.ca for CWRC-Writer objects.

Can we copy that for the DToC bridge module, @nikathone ?

**Comment by ilovan** [comment]
Example of CWRC_Document object:
https://beta.cwrc.ca/islandora/object/cwrc%3A792bf5fd-fea4-4c1a-aad1-b7fd29df6eb0

It looks like this setting hasn't been pushed to beta at all by Echidna. Hold of on this, @nikathone , until I figure out what's going on.

**Comment by nikathone** [comment]
Is it on staging?

**Comment by ilovan** [comment]
nope, it's no longer anywhere, though it was at the time tested and vetted on both dev-03 and test-01. Hang on.
